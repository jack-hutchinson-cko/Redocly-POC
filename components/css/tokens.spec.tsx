import * as tokens from '../index';

describe('tokens', () => {
	for (let key in tokens) {
		test(key, () => {
			expect.assertions(1);
			expect(tokens[key]).toMatchSnapshot();
		});
	}
});
