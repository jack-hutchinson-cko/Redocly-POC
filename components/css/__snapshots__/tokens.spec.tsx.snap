// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`tokens ButtonLink 1`] = `[Function]`;

exports[`tokens Card 1`] = `[Function]`;

exports[`tokens Information 1`] = `[Function]`;

exports[`tokens Link 1`] = `[Function]`;

exports[`tokens ThemeToggle 1`] = `[Function]`;
